sandbox

```diff
@@ -31,6 +31,25 @@ export INDEXER_BRANCH=""
 export INDEXER_SHA=""
 export INDEXER_DISABLED=""
 
+
+
+######################################
+# Compatibility with Windows / MSYS2 #
+######################################
+
+# All commands that use a TTY/pseudo-TTY
+# need to be prefixed by "$WINDOWS_PTY_PREFIX"
+WINDOWS_PTY_PREFIX=
+if [ "$(uname)" == "Darwin" ]; then
+  true
+elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ] || \
+     [ "$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
+  WINDOWS_PTY_PREFIX="winpty"
+fi
+
+
+
+
```

Comment:
Just remove a little bit of whitespace, otherwise looks great
```suggestion
######################################
# Compatibility with Windows / MSYS2 #
######################################

# All commands that use a TTY/pseudo-TTY
# need to be prefixed by "$WINDOWS_PTY_PREFIX"
WINDOWS_PTY_PREFIX=
if [ "$(uname)" == "Darwin" ]; then
  true
elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ] || \
     [ "$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
  WINDOWS_PTY_PREFIX="winpty"
fi

```